README.md

```diff
@@ -879,5 +879,5 @@ _all admin calls require auth and admin privilege_
 * User can edit comments in 5 mins (configurable) window after creation.
 * User ID hashed and prefixed by oauth provider name to avoid collisions and potential abuse.
 * All avatars resized and cached locally to prevent rate limiters from oauth providers, part of [go-pkgz/auth](https://github.com/go-pkgz/auth) functionality.
-* Images can be proxied (`IMG_PROXY=true`) to prevent mixed http/https.
+* Images can be proxied (`IMAGE_PROXY_HTTP2HTTPS=true`) to prevent mixed http/https. Beware, images which are posted with this parameter enabled will be served from proxy even when it will be disabled.
```

Comment:
I don't get it. http2https doesn't serve anything "from proxy", all it does it reverse proxy and serves from the original location unless CACHE_EXTERNAL enabled

Reply:
Adjusted readme to reflect that.